src/main/java/edu/harvard/iq/dataverse/FilePage.java

```diff
@@ -544,23 +544,29 @@ public String ingestFile() throws CommandException{
     }
 
     public String uningestFile() throws CommandException {
-        
+
         if (!file.isTabularData()) {
-            if(file.isIngestProblem()) {
-                User u = session.getUser();
-                if(!u.isAuthenticated() ||  !(permissionService.permissionsFor(u, file).contains(Permission.PublishDataset))) {
-                    logger.warning("User: " + u.getIdentifier() + " tried to uningest a file");
-                    //Shouldn't happen (choice not displayed for users who don't have the right permission), but check anyway
-                    JH.addMessage(FacesMessage.SEVERITY_WARN, BundleUtil.getStringFromBundle("file.ingest.cantUningestFileWarning"));
-                    return null;
-                }
-              file.setIngestDone();
-              file.setIngestReport(null);
+            //Ingest never succeeded, either there was a failure or this is not a tabular data file
+            User u = session.getUser();
+            if (!u.isAuthenticated() || !u.isSuperuser()) {
```

Comment:
OK, I thought what you had here before was on purpose - to give  the dataset author a way to remove an ingest failure report - no?

Reply:
(those are only visible to the owners/authors, so not a big deal either way)

Reply:
Would be nice if that's OK. (Probably less work for superusers as that warning bothers the authors.) 
At QDR all of this is with the publish permission.